index.js

```diff
@@ -20,17 +22,17 @@ module.exports = function(source) {
   });
 
   const callback = this.async();
-  postcss([ ...pluginsInit, postcssKremling() ])
+  postcss([ ...pluginsInit, postcssKremling(kremlingNamespace)() ])
     .process(source, {
       ...restOfOptions,
       to: './',
       from: './',
     })
     .then((result) => {
       if (result.css) {
-        callback(null, `module.exports = { styles: '${result.toString().replace(/(\s)+/g, ' ')}', id: '${kremId.id}', ${kremlingNamespace} };`);
+        callback(null, `module.exports = { styles: '${result.toString().replace(/(\s)+/g, ' ').replace(/'/g, `\\'`)}', id: '${kremId.id}', ${kremlingNamespaceString} };`);
```

Comment:
why are we replacing with a space character instead of empty string? Would empty string cause the css string to become invalid?